packages/ui/src/components/Dialog/Dialog.tsx

```diff
@@ -125,51 +119,58 @@ DialogClose.displayName = DialogPrimitive.Close.displayName;
 const DialogHeader = ({
   className,
   size,
+  children,
+  padding = true,
   ...props
-}: React.HTMLAttributes<HTMLDivElement> & SizeProp) => (
-  <>
-    <DialogClose position="right" size={size}>
-      <IconButton name="cross" variant="ghost" />
+}: DialogProps & SizeProp & PropsWithChildren) => (
+  <div
+    className={twMerge(
+      "flex items-center justify-between",
+      padding && defaultPaddingClassName,
```

Comment:
This feels weird to me, to add a `padding` optional prop, that is a boolean, to display (or not) default padding values.

Why are we doing this approach instead of letting the `className` override the default class at line 128, where we could get the default padding values?

Reply:
yeah, will remove this, with twMerge it's easy to override classes.